data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -0,0 +1,549 @@
+              available to this session - this indicates
+              movement larger than a single pixel.
+
+              For example, a fast movement against a barrier on the right edge
+              of a screen may logically put the cursor dozens of pixels into
+              the (non-existing) screen on the other side of the barrier.
+              It is the application's responsibility to adjust the cursor
+              position as necessary.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>barrier_id u</term>
+            <listitem><para>
+              The barrier id of the barrier that triggered. If the value is
+              nonzero, it matches the barrier id as specified in
+              org.freedesktop.portal.InputCapture.SetPointerBarriers().
+
+              If the id is zero, the pointer barrier could be determined.
+              If the id is missing, the input capture was not triggered by a
+              pointer barrier.
+
+              Where more than one pointer barrier are triggered by the same
+              movement it is up to the compositor to choose one barrier (or use
+              a barrier id of zero).
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <signal name="Activated">
+      <arg type="o" name="session_handle" direction="out"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+    </signal>
+
+    <!--
+        Deactivated:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+
+        The Deactivated signal is emitted when input capture stopped and input events
+        are no longer sent to the application. To prevent future input
+        capture, an application must call org.freedesktop.portal.InputCapture.Disable().
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>serial u</term>
+            <listitem><para>
+                The same serial number as in the corresponding
+                #org.freedesktop.portal.InputCapture::Activated signal.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>cursor_position dd</term>
+            <listitem><para>
+              The current cursor position. Note that this position may be
+              outside the Zones available to this session - this indicates
+              movement larger than a single pixel.
+
+              For example, a fast movement against a barrier on the right edge
+              of a screen may logically put the cursor dozens of pixels into
+              the (non-existing) screen on the other side of the barrier.
+              It is the application's responsibility to adjust the cursor
+              position as necessary.
+            </para></listitem>
```

Comment:
@jadahl  - this is clearly copy/paste but I'm wondering whether there's use-case for including the cursor position on the `Deactivated` signal. Seems superfluous since the only thing the client can do is paint the cursor where it left, and it's unlikely to stay there anyway.

Reply:
Removed in the latest version